pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -480,6 +480,11 @@ message CloudStorageTestSpec {
   // NEXT ID: 3;
 }
 
+message TableReplicationMetadata {
+  optional cockroach.sql.sqlbase.TableDescriptor source_descriptor = 1 [(gogoproto.nullable) = false];
+  optional string database_prefix = 2 [(gogoproto.nullable) = false];
```

Comment:
nit: i'd rename this to parent_database_name

also i think we still need to plumb the schema name field here, just to add to the dlq table _name_ to prevent name collisions. For example: suppose the user has the following tables: `mydb.schema_foo.baz` and `mydb.schema_bar.baz`, i think we want to create a unique dlq table for each with the table names: `schema_foo_baz` and `schema_bar_baz`.

Reply:
re: dlq table name comments, should we settle on this naming convention: `db.crdb_replication.schema_table`?

Reply:
yeah, that's my preference

Reply:
yeah, that's my preference

Reply:
yeah, that's my preference

Reply:
yeah, that's my preference